### PR TITLE
Add openj9-openjdk-notices

### PR DESCRIPTION
**Pull request by andrew-m-leonard**
Signed-off-by: andrew-m-leonard <andrew_m_leonard@uk.ibm.com>

### CODE REVIEW

**Comment by andrew-m-leonard** [comment]
Copyright CI issue as legal file has no copyright

File: openj9-openjdk-notices [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  The OpenSSL toolkit stays under a double license, i.e. both the conditions of
+  the OpenSSL License and the original SSLeay license apply to the toolkit.
+  See below for the actual license texts.
+
+  OpenSSL License
+  ---------------
+
+/* ====================================================================
+ * Copyright (c) 1998-2018 The OpenSSL Project.  All rights reserved.
+ *
```
Comment (keithc-ca) [inline openj9-openjdk-notices:21]: Could you please trim the trailing whitespace and remove the final blank line?

**Review by keithc-ca — COMMENTED** [review]
I think the licenses should be present as plain text, as opposed to being embedded in C-style comments.

**Comment by andrew-m-leonard** [comment]
> I think the licenses should be present as plain text, as opposed to being embedded in C-style comments.

The current format is keeping in the exact copy of the source: https://www.openssl.org/source/license-openssl-ssleay.txt

**Comment by andrew-m-leonard** [comment]
Re-copied latest license from openssl.org, and removed trailing line...

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**